### DISCUSSION TITLE
Q: can this be used with pi Pico w?

### DISCUSSION THREAD

**Discussion by VilliVar** [Berny23/LD-ToyPad-Emulator#81] [Q&A] [answered]
I am not too knowledgeable about raspberry pi's, so I don't know if this is possible. I am trying to get this set up for my ps5 and don't want to spend too much, any help would be greatly appreciated

**Comment by lazzer2000** [comment]
I'm curious about this as well since the Pi zero is Spendy, but you can get the pi pico w for like $14.

**Comment by Stealthanator1** [comment]
I would also like to know this as the Pico w is so much cheaper

**Comment by dverzolla** [comment]
Banana Pi Zero can be a cheaper alternative.

**Reply by lazzer2000** [reply]
And the process would be the same on that?

**Reply by dverzolla** [reply]
I have not tested yet, but should be the same. 

Waiting my Banana arrives to start testing. 

Will post here the results.

**Comment by Berny23** [comment, marked as answer]
No, the Raspberry Pi Pico (W) is a microcontroller, not a single board computer (SBC).

Just look at the specs, you cannot run Linux on 256 KB RAM with 2 MB storage!

**Comment by DomanDragon** [comment]
I thought the same thing, until I realized that there was no MicroSD card slot, then I realized that microcontrollers and microcomputers are different.

I ended up buying a used NanoPi NEO 1, and while I had to get a Wi-Fi adapter for it, it worked perfectly using the [DietPi](https://dietpi.com/) distro. I'd recommend finding a NanoPi board, preferably one with Wi-Fi so you don't have to plug it into your router to use it, and using that. I've also heard BananaPi works just as well. (I tried a ClockworkPi, by the way, but that didn't quite work out.)

Basically: if it supports both Wi-Fi and USB On-The-Go, has an SD card slot of some kind, and has a minimum of 256GB RAM, it'll run this project fine.

**Reply by cort1237** [reply]
> and has a minimum of 256GB RAM, it'll run this project fine.

256GB RAM is quite a beefy requirement, dont you think? lol
